frontend/packages/ux-editor/src/containers/DesignView/AddItem/FeedbackForm.tsx

```diff
@@ -0,0 +1,61 @@
+import type { ReactElement } from 'react';
+import { submitFeedbackPath } from 'app-shared/api/paths';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { post } from 'app-shared/utils/networking';
+import { FeedbackFormImpl } from '@studio/feedback-form';
+import { toast } from 'react-toastify';
+
+/**
+ * This is a feedback form to gather feedback on the new design for adding components.
+ * It uses the FeedbackForm component from the @studio/feedback-form package.
+ * The form is temporary, and will be removed once the new design is fully tested, or we decide to go in a different direction.
+ * As such, all texts are hardcoded in Norwegian, to avoid adding unnecessary translations.
+ * @returns The FeedbackForm component.
+ */
+export function FeedbackForm(): ReactElement {
+  const { org, app } = useStudioEnvironmentParams();
+
+  const submitFeedback = async (answers: Record<string, string>) => {
+    try {
+      // Using regular axios post rather than a mutation hook, since we are not storing
+      // the feedback in the cache, nor are we updating any state.
+      await post(submitFeedbackPath(org, app), { answers: { ...answers } });
+      toast.success('Takk for tilbakemeldingen!');
```

Comment:
Now that you have added some tests, maybe we should add tests for the final ones as well? 😄

Reply:
Let's maybe do that in a separate PR?